cpp/models/abm/migration_rules.cpp

```diff
@@ -18,17 +18,17 @@
 * See the License for the specific language governing permissions and
```

Comment:
local mobility is misleading, since in the context of the ABM, they are not local at all. We already use the terms local and global in the abm, local means for a single location, global is for the whole (ABM) world.